pkgs/kswitch/kswitch.sh

```diff
@@ -291,6 +390,15 @@ while (( "$#" )); do
         jsonOutput=1
         shift
         ;;
+        --force-refresh)
+        forceRefresh=1
+        shift
+        ;;
+        --force-unlock)
+          force_unlock
+          exit 0
```

Comment:
`--force-unlock` is a bit cryptic, and in reality this cleans creds cache

Reply:
keeping the function, but only remove locks